### ISSUE TITLE
Invalid Scopes: user_photos, user_posts, user_videos.

### ISSUE THREAD

**Issue by Vincy69** [fbsamples/live-comments-teleprompter#17]
Hi,

I'm getting this error message when trying to connect : 

"Invalid Scopes: user_photos, user_posts, user_videos. This message is only shown to developers. Users of your app will ignore these permissions if present. Please read the documentation for valid permissions at: https://developers.facebook.com/docs/facebook-login/permissions"

But, I'm able to log in when my application is in development mode, but it won't recognize any facebook post url.

If you have any idea, thanks for your help !

**Comment by zenlita** [comment]
Hi there, just to clarify, could you share a Facebook post url that you were trying? 
For it to work it must be a public URL. 

Thanks!

**Comment by Vincy69** [comment]
Hi, thanks for your quick reply, here's an url I'm trying to work with for exemple : 

https://www.facebook.com/elephorm/videos/10156942970449638/

**Comment by zenlita** [comment]
Thanks for the url! I can see this post is public =) Could you check your developer tools console (inspector) if there is any error msg? One common error would be that your access token may have expired.

**Comment by Vincy69** [comment]
I'm getting this message in the console :

**Comment by zenlita** [comment]
Thanks for the URL! I've tested with my implementation and I could grab comments from this post. Just throwing some ideas out, make sure that your access token has 'manage_pages' permission.

**Comment by Vincy69** [comment]
I'm getting this message :
`{message: “(#10) To use ‘Page Public Content Access’, your us…https://developers.facebook.com/docs/apps/review.“, type: “OAuthException”, code: 10, fbtrace_id: “HfgU6fS24dI”}
code
:
10
fbtrace_id
:
“HfgU6fS24dI”
message
:
“(#10) To use ‘Page Public Content Access’, your use of this endpoint must be reviewed and approved by Facebook. To submit this ‘Page Public Content Access’ feature for review please read our documentation on reviewable features: https://developers.facebook.com/docs/apps/review.”
type
:
“OAuthException”
__proto__
:
Object`

Si I'm thinking I need to get an approval from facebook for each : user_post , video, and photo ...

**Comment by zenlita** [comment]
Yeah....that seems like it. Could you request for review? If I find any other way I will update here.

**Comment by zenlita** [comment]
Hi there! I believe you need to get your app approved + business verification done to get the permissions required. I looked back to what I did and realize that was what I needed to do earlier.

**Comment by Vincy69** [comment]
Still waiting to get my app approved, I'll get back to you. Thanks for your help guys !

**Comment by webmagnets** [comment]
> Still waiting to get my app approved, I'll get back to you. Thanks for your help guys !

Did this start working after your app got approved?

**Comment by JensBergander** [comment]
I can't login. what I do wrong. I have the same issue :-/

**Comment by 1v4r** [comment]
Hi!
I'm having the same issue. Could it bee facebook has changed their permission structure so that this app no longer works?
What kind of permissions do I have to enable?
Do I have to request any advanced permissions?
